modules/serverless-sinkbinding-odc.adoc

```diff
@@ -0,0 +1,52 @@
+[id="serverless-sinkbinding-odc_{context}"]
+= Creating a sink binding using the Developer perspective
+
+You can create and verify a basic sink binding from the {product-title} web console.
+
+.Prerequisites
+
+To create a sink binding using the *Developer* perspective, ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Create a Knative service to use as a sink:
+.. In the *Developer* perspective, navigate to *+Add* -> *YAML*.
+.. Copy the example YAML:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+.. Click *Create*.
+
+. Create a sink binding in the same namespace as the service created in the previous step, or any other sink that you want to send events to.
+
+.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*. The  *Event Sources* page is displayed.
+.. Optional: If you have multiple providers for your event sources, select the required provider from the *Providers* list to filter the available event sources from the provider.
+.. Select *Sink Binding* and then click *Create Event Source*. The *Create Event Source* page is displayed.
+.. In the *apiVersion* field enter `v1`.
+.. In the *Kind* field enter `Event`.
+.. Select a *Sink*. This can be either a *Resource* or a *URI*. In this example, the `event-display` service created in the previous step is used as the *Resource* sink.
+.. Click *Create*.
+
+.Verification
+
+You can verify that the sink binding was created and is connected to the sink by viewing the *Topology* page.
+
+. In the *Developer* perspective, navigate to *Topology*.
+. View the sink binding and sink.
++
+image::verify-sinkbinding-odc.png[View the sink binding and service in the Topology view]
```

Comment:
I'm also attaching a better image that actually shows data received by event-display pod, and successes on cronjob.

![verify-sinkbinding-odc](https://user-images.githubusercontent.com/540893/132219270-bf279c26-ac67-4ecc-8e3f-4f322f96e091.png)